make_all_metro_poly.py

```diff
@@ -31,5 +34,19 @@ def make_disjoint_metro_polygons():
     print("END")
 
 
+def main() -> None:
+    parser = argparse.ArgumentParser()
```

Comment:
NIT: I suggest you to start using https://pypi.org/project/click/

Reply:
Good idea for a separate PR.